### ISSUE TITLE
Add new physics expression that returns the angle the object is moving: LinearVelocityAngle()

### ISSUE THREAD

**Issue by tristanbob** [4ian/GDevelop#4338]
## Description

When using forces, users can know the speed and direction of an object using:
- ForceLength() 
- ForceAngle()

However, when using Physics, users can only know the speed, but not the direction
- LinearVelocity()

## Solution suggested

- [ ] Add new expression `VelocityAngle` or `AngleOfVelocity` that returns the angle the direction is moving from Physics

**Comment by Bouh** [comment]
Even Linear Speed is deprecated.

![image](https://user-images.githubusercontent.com/1670670/194288337-5542fb59-bfb7-4b4b-88fc-74750b21010d.png)

**Comment by Bouh** [comment]
And for Physics 2.0 , there is `BluePlayerShip1.Physics2::AngularVelocity()`

**Comment by tristanbob** [comment]
> Even Linear Speed is deprecated. because it's not Physic 2.0.
> 
> ![image](https://user-images.githubusercontent.com/1670670/194288337-5542fb59-bfb7-4b4b-88fc-74750b21010d.png)

Linear speed is still offered in Physics 2.0:  `RedBarrel.Physics2::LinearVelocity()`

Angular speed is rotation speed, not movement angle.  `BluePlayerShip1.Physics2::AngularVelocity()`

**Comment by tristanbob** [comment]
Here is my strategy for implementing this. Let me know if this is the right way to go.  :)

GDevelop\Extensions\Physics2Behavior\JsExtension.js
```
      aut
      .addCondition(
        'LinearVelocityAngle',
        _('Linear velocity angle'),
        _('Test an object linear velocity angle.'),
        _('the linear velocity angle'),
        _('Velocity'),
        'res/physics32.png',
        'res/physics32.png'
      )
      .addParameter('object', _('Object'), '', false)
      .addParameter('behavior', _('Behavior'), 'Physics2Behavior')
      .useStandardRelationalOperatorParameters(
        'number',
        gd.ParameterOptions.makeNewOptions().setDescription(
          _('Angle to compare to (in degrees)')
        )
      )
      .getCodeExtraInformation()
      .setFunctionName('getLinearVelocityAngle');

    aut
      .addExpression(
        'LinearVelocityAngle',
        _('Linear velocity angle'),
        _('Get the linear velocity angle of an object.'),
        _('Velocity'),
        'res/physics32.png'
      )
      .addParameter('object', _('Object'), '', false)
      .addParameter('behavior', _('Behavior'), 'Physics2Behavior')
      .getCodeExtraInformation()
      .setFunctionName('getLinearVelocityAngle');
```
GDevelop\Extensions\Physics2Behavior\physics2runtimebehavior.ts
```
    getLinearVelocityAngle(): float {
      // If there is no body, set a new one
      if (this._body === null) {
        if (!this.createBody()) return 0;
      }
      const body = this._body!;

      // Get the linear velocity angle
       return Math.atan2(
        body.GetLinearVelocity().get_y() * this._sharedData.scaleY
        body.GetLinearVelocity().get_x() * this._sharedData.scaleX,
      );
    }
 ```

**Comment by tristanbob** [comment]
This was fixed by #4798.
